### DISCUSSION TITLE
Need XCH to Join Pool

### DISCUSSION THREAD

**Discussion by neerajvirodhiya** [Chia-Network/chia-blockchain#7282] [Support]
Hi Team, This is an my chia address "xch1vafmhpa7mkqa45dercq37gz2wxyhhhxlxf6qn2sykkp0hm0pfxcqxkkaks"
Yesterday I have plotted 4 plot K32. and starts farming then i click to join pool then i saw message i need some xch from faucets to join pool and i click on https://faucet.chia.net/ and add my address and click submit. But i did not received faucets. After few hours i checked again on https://faucet.chia.net then it showing already sent. But my wallet is empty. Please tell me when i get faucets?

**Comment by ardikars** [comment]
https://chialinks.com/faucets/

**Reply by loppefaaret** [reply]
also make sure your wallet is fully synced! it should display the same/near height as the full node is reporting

**Reply by neerajvirodhiya** [reply]
@loppefaaret  today I have created 4 plots. How much time it take to sync ? And when is wallet fully sync??

**Reply by loppefaaret** [reply]
wallet is fully synced when it reaches the same height as the full node

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
